e2e/tests/ui-driven/src/create-flow-with-geospatial.spec.ts

```diff
@@ -105,6 +109,25 @@ test.describe("Flow creation, publish and preview", () => {
     await answerFindProperty(page);
     await clickContinue({ page });
 
+    await expect(
+      page.getByRole("heading", { name: "About the property" }),
+    ).toBeVisible();
+
+    // Check map component has geoJson content
+    await checkGeoJsonContent(page, mockMapGeoJson);
+
+    // Check property info is being shown
+    await expect(page.getByText("Test Street, Testville")).toBeVisible();
+    await expect(page.getByText("Residential - Semi Detached")).toBeVisible();
+    const changeButton = page.getByRole("button", {
+      name: "Change your Property type",
+    });
+
+    await changeButton.click();
+
+    await expect(page.getByText("Unknown")).toBeVisible();
```

Comment:
I don't think this "Unknown" error is actually meaningful to test here? 

The "change" link relies on the Preview store method `overrideAnswer` and notably _at least one node located before PropertyInformation that sets passport value `property.type`_ in order to successfully navigate as explained by other existing frontend tests (in "real" flows, this would be the entire `opensystemslab/property-types` portal).

I'd suggest:
- Adding "PropertyInformation" to the E2E tests _without_ the "change" link toggled on and confirming it's not visible (and relying on our existing editor test rather than e2e tests for navigation)
- Or correctly mocking this full journey, which is roughly:
  - The flow needs to create at least one node (Question or Checklist) that sets `property.type` in a location _before_ the PropertyInformation component and _after_ FindProperty (so it's auto-answered and never seen by user _unless_ the user clicks "change")
    - The node should have option passport values that are less-granular than the fetched property type from OS (eg `residential`, `commercial`) 
  - When you click "Change", we should test that you navigate _back_ to this node successfully and the `residential` option is auto-filled (eg not `Unknown` error !!)
  - You can successfully "change" the answer to another option
  - When you "Continue" forwards back to PropertyInformation, your new selection displays in type field

Reply:
This makes a lot more sense than the current way I was using it. I'll update the tests